ldk/go/filesystemClient.go

```diff
@@ -289,14 +289,10 @@ func (f *GRPCFile) Close() error {
 		},
 	})
 	if err != nil {
+		println("[ERROR] GRPCFile.Close - Received error sending Close", err.Error())
 		return err
 	}
-	select {
-	case <-f.ctx.Done():
```

Comment:
Do we know why we weren't receiving either of these messages?

Reply:
So ... the [ServerStream](https://pkg.go.dev/google.golang.org/grpc#ServerStream) doesn't give us a mechanism to explicitly cancel the stream; instead the Server returns from the method and that is how the stream is closed.

It doesn't look like the GRPC implementation gives clients in Bidirectional Streaming calls a mechanism to know a call is closed without trying to receive one last message and receiving `io.EOF`.

Reply:
Could I theoretically exhaust my available file handles by running `Open` in a loop then?

Reply:
Not the file handles themselves; those are `defer file.Close()` upon exit from the server's loop.